docs/src/lib/config.ts

```diff
@@ -41,7 +41,7 @@ export const SIDEBAR = {
   ],
   Usage: [
     { text: "Integration", link: "/en/usage/integration" },
-    { text: "Contribution", link: "/en/usage/contribution" },
+    { text: "Contribution", link: "/en/usage/contribution" }, { text: "Migration", link: "/en/usage/migration" },
```

Comment:
format

Reply:
@mrivasperez add a return here to match the formatting and also update the naming and file naming as well. Overall I like this content and would like to get it merged in.